receiver/jmxreceiver/config.go

```diff
@@ -95,10 +96,21 @@ func (oec otlpExporterConfig) headersToString() string {
 }
 
 func (c *Config) parseProperties() []string {
-	parsed := make([]string, 0, len(c.Properties))
-	for property, value := range c.Properties {
-		parsed = append(parsed, fmt.Sprintf("-D%s=%s", property, value))
+	parsed := make([]string, 0, 1)
+	if len(c.ResourceAttributes) > 0 {
+		attributes := make([]string, 0, len(c.ResourceAttributes))
+		for k, v := range c.ResourceAttributes {
+			attributes = append(attributes, fmt.Sprintf("%s=%s", k, v))
+		}
+		parsed = append(parsed, fmt.Sprintf("-Dotel.resource.attributes=%s", strings.Join(attributes, ",")))
+	}
+
+	logLevel := "info"
+	if len(c.LogLevel) > 0 {
+		logLevel = strings.ToLower(c.LogLevel)
```

Comment:
Can we use a switch to ensure it is one of the enumerated log levels?

Reply:
That validation is done on line 168, in the validate function

Reply:
Thanks, missed that